File: examples/indexer/python/search_tx_address_sigtype.py
# search_tx_address_sigtype.py
import json
# requires Python SDK version 1.3 or higher
from algosdk.v2client import indexer

myindexer = indexer.IndexerClient(
    indexer_token="", indexer_address="http://localhost:8980")

response = myindexer.search_transactions_by_address(
    address="XIU7HGGAJ3QOTATPDSIIHPFVKMICXKHMOR2FJKHTVLII4FAOA3CYZQDLG4", sig_type="msig")

print("sig_type: msig = " +
      json.dumps(response, indent=2, sort_keys=True))

File: examples/indexer/javascript/SearchTxAddresssigtype.js
// SearchTxAddresssigtype.js
// requires algosdk@1.6.1 or higher 
// verify installed version
// npm list algosdk

const algosdk = require('algosdk');
const indexer_token = "";
const indexer_server = "http://localhost";
const indexer_port = 8980;

// Instantiate the indexer client wrapper
let indexerClient = new algosdk.Indexer(indexer_token, indexer_server, indexer_port);
    
(async () => {
    let address = "XIU7HGGAJ3QOTATPDSIIHPFVKMICXKHMOR2FJKHTVLII4FAOA3CYZQDLG4";
    let sig_type = "msig"; 
    let response = await indexerClient.searchForTransactions()
        .address(address)
        .sigType(sig_type).do();
    console.log("Information for Transaction search: " + JSON.stringify(response, undefined, 2));
    }
   
)().catch(e => {
    console.log(e);
    console.trace();
});